station-flattening-v0/src/main/java/net/modificationstation/stationapi/mixin/flattening/client/MixinBaseClientInteractionManager.java

```diff
@@ -0,0 +1,25 @@
+package net.modificationstation.stationapi.mixin.flattening.client;
+
+import net.minecraft.client.BaseClientInteractionManager;
+import net.minecraft.client.Minecraft;
+import org.spongepowered.asm.mixin.Final;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.Shadow;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.ModifyArgs;
+import org.spongepowered.asm.mixin.injection.invoke.arg.Args;
+
+@Mixin(BaseClientInteractionManager.class)
+public class MixinBaseClientInteractionManager {
+	@Shadow @Final protected Minecraft minecraft;
+	
+	@ModifyArgs(method = "method_1716", at = @At(
```

Comment:
Couldn't this be done with a `@ModifyConstant` on 256 to preserve the `meta, id` order instead of making in `id, meta`?